script/fetch_content.py

```diff
@@ -5,28 +5,45 @@
 
 
 def fetch_arguments():
+  """Fetch arguments from command line"""
+
   parser = argparse.ArgumentParser()
-  parser.add_argument('--url', help='add url')
-  parser.add_argument('--text', help='add text')
-  args = parser.parse_args() 
-  return args
+  parser.add_argument('url', help='add url')
+  parser.add_argument('text', help='add text')
+  args = parser.parse_args()  
+  return (args.url, args.text)
+
+
+def print_10_characters_before_and_after_text(text, r):
+  """
+  Print text with  additional characters.
+
+  This function retrieves the text and returns it with an additional ten characters before and after the text.
+
+  parameters:
+  text: text that someone enters
+  r.text: all text on the website
+  start: index of the first letter of the entered text
+  end: index of the last letter of the entered text
+  new_text: a string with additional characters added before start and after end.
 
+  Return:
+  string: text with 10 additional characters before and after.
+  
+  """
 
-def print_10_characters_before_and_after_text():
   for m in re.finditer(text, r.text):
```

Comment:
If you only want to find first occurrence then for loop is not good solution.
Better use if.
Also you need to provide return values if the string is not found in the text. You can return empty string then.

Reply:
I don't know how to do it with the if statement, re.finditer () doesn't work with "if"

Reply:
finditer() is not the only function which you can use to search string.

But. If you want to use it then few hints:
 * finditer() returns `iterator`
 * Iterator has function `next()` which you can use to get next found match.

Google `python iterator` for more info.